llpc/test/shaderdb/core/OpSDotAccSat_TestIVec16bit_lit.spvasm

```diff
@@ -0,0 +1,63 @@
+; BEGIN_SHADERTEST
```

Comment:
nit: no need for `_lit` in the test name. Also applies to other test files.

Reply:
OK. Accept.